rust/connlib/snownet/src/node.rs

```diff
@@ -776,6 +773,11 @@ where
             }
         }
     }
+
+    /// Sample a relay to use for a new connection.
+    fn sample_relay(&mut self) -> Option<RId> {
+        self.allocations.keys().copied().choose(&mut self.rng)
+    }
```

Comment:
Instead of sampling randomly, we can in the future also experiment with picking the one with the lowest RTT for example or any other metric that we'd like and is measureable from the clients.

Reply:
The good thing of randomness is that there is at least a chance that a connection works if yoi retry and one relay is faulty for some reason.

Also, with in order, do you mean round-robin? Only ever use the first? When would I use the others?